eng/Workarounds.props

```diff
@@ -29,4 +29,11 @@
     <!-- Ignore warning about calling the Pack target on Web SDK projects. Our build scripts call /t:pack on everything in this repo. -->
     <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
   </PropertyGroup>
+
+  <!-- The SDK includes some conflict resolution targets that check for duplicate items in the publish targets. It's very greedy
+       and ends up marking the same files in different projects as duplicates of each other. We disable this check here to work
+       around this issue. -->
+  <PropertyGroup>
+    <ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles>
```

Comment:
We bumped the version of the SDK we are one and picked up a change to the conflict resolution targets mention here so this workaround is necessary since this is new code.

Reply:
This only affects a couple of projects, particularly the test sites that we use which end up having a lot of common files (wwwroot/style.css for example) which trigger this buggy code path. I'm working on opening a bug in the SDK repo to see if there is an issue with this.